stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/requests/PinnedMessagesRequest.kt

```diff
@@ -0,0 +1,77 @@
+package io.getstream.chat.android.client.api2.model.requests
+
+import com.squareup.moshi.JsonClass
+import io.getstream.chat.android.client.api.models.PinnedMessagesPagination
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.models.Message
+import java.util.Date
+
+@JsonClass(generateAdapter = true)
+internal data class PinnedMessagesRequest(
```

Comment:
Could probably have some docs here. 

For the API/Client/Interfaces etc I guess it's okay if we only have it in one place - the core part of the client, but this seems like it would be great to have some docs for the properties.

Maybe we can open a TODO to add it, if we're time-stricken.